src/components/ModalWindow/ModalWindow.tsx

```diff
@@ -61,16 +76,16 @@ const ModalWindow: FC = () => {
               <Form.Control
                 className={s.formControl}
                 type="datetime-local"
+                value={input.date}
                 required
-                placeholder="start data"
                 onChange={(e: any) => handleChange(e, "date")}
```

Comment:
Why any if you have type in handleChange function

Reply:
I think about not repeating 2 times ChangeEvent<HTMLInputElement> . Fixed